example-crates/bitmex/guest/Cargo.toml

```diff
@@ -3,12 +3,13 @@ name = "bitmex_guest"
 version = "0.1.0"
 edition = "2021"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
+network_tests = []
 
 [dependencies]
 carol_guest = { workspace = true }
 bincode = { workspace = true }
 time = { version = "0.3", features = ["std", "serde-human-readable", "macros", "parsing"] }
 url = "2"
 serde_json = "1"
-serde = {  version = "1", features = ["derive"] }
+serde = { workspace = true }
```

Comment:
`carol_guest` should probably re-export this. Not relevant for this PR.

Reply:
yeah that makes more sense